K8s-deployment/Charts/redis/redis-autoscaler.sh

```diff
@@ -32,15 +32,15 @@ do
             master_id_line=$(expr $line_num + 2)
             master_id=$(redis-cli --cluster check $pod_name.redis-redis-cluster-headless:6379 -a $REDIS_PASSWORD |awk 'NR=='$master_id_line' {print $2}')
             echo "Master id to replicate: $master_id"
-            helm repo add bitnami https://raw.githubusercontent.com/bitnami/charts/43174e1970616584f893f66c1fbcac00d110c633/bitnami/
+            helm repo add bitnami https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
             helm upgrade --timeout 600s --reuse-values redis --set "password=$REDIS_PASSWORD,cluster.update.addNodes=true,cluster.update.currentNumberOfNodes=$current_num_nodes,cluster.nodes=$new_num_nodes,updateJob.nodeType=slave,updateJob.masterID=$master_id" bitnami/redis-cluster -n redis
```

Comment:
Upgrade helm redis release version in autoscaler. Also increase image ver of autoscaler, and push the autoscaler image to registry.

Reply:
Update docker image build in readme  with newer version image